From 50d47d840a6d34bf7994bcaf11bf0e2e94c0583d Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@iodev.co.uk>
Date: Fri, 01 Jul 2022 16:08:05 +0200
Subject: [PATCH] Fix inclusion of signal.h

POSIX.1-1988 already specifies it as <signal.h>, <sys/signal.h> is an
artifact of non-conformance.

Signed-off-by: Ismael Luceno <ismael@iodev.co.uk>
Upstream-Status: Pending
---
 src/diskseekd.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/diskseekd.c b/src/diskseekd.c
index 324d091e9f89..cde35e8ff019 100644
--- a/src/diskseekd.c
+++ b/src/diskseekd.c
@@ -16,7 +16,7 @@
 #include <sys/stat.h>
 #include <linux/major.h>
 #include "enh_options.h"
-#include <sys/signal.h>
+#include <signal.h>
 #include <errno.h>
 
 int eioctl(int fd, int command,void * param, char *emsg)
